xml/admin_nfsganesha.xml

```diff
@@ -39,403 +26,134 @@
-     <literal>rados://&lt;POOL&gt;/&lt;NAMESPACE&gt;/&lt;OBJECT&gt;</literal>,
-     where <literal>POOL</literal> is the &rados; pool where the object is
-     stored, <literal>NAMESPACE</literal> the pool namespace where the object
-     is stored, and <literal>OBJECT</literal> the object name.
-    </para>
-    <para>
-     To support the &dashboard;'s &ganesha; management capabilities, you need
-     to follow a convention on the name of the &rados; object for each service
-     daemon. The name of the object must be of the form
-     <literal>conf-<replaceable>MINION_ID</replaceable></literal> where
-     MINION_ID corresponds to the &sminion; ID of the node where this service
-     is running.
-    </para>
-  </sect2>
-   <sect2 xml:id="ganesha-nfsport">
-    <title>Changing Default &ganesha; Ports</title>
-    <para>
-     &ganesha; uses the port 2049 for NFS and 875 for the rquota support by
-     default. To change the default port numbers, use the
-     <option>NFS_Port</option> and <option>RQUOTA_Port</option> options inside
-     the <literal>NFS_CORE_PARAM</literal> section, for example:
-    </para>
-<screen>
-NFS_CORE_PARAM
-{
-NFS_Port = 2060;
-RQUOTA_Port = 876;
-}
-</screen>
-</sect2>
-
-  <sect2 xml:id="ceph-nfsganesha-config-exports-general">
-   <title>Exports Configuration</title>
+ <sect1 xml:id="ceph-nfsganesha-nfservice">
+   <title>Create an NFS Service</title>
    <para>
-    Exports configuration is stored as &rados; objects in the &ceph; cluster.
-    Each export block is stored in its own &rados; object named
-    <literal>export-<replaceable>ID</replaceable></literal>, where
-    <replaceable>ID</replaceable> must match the <literal>Export_ID</literal>
-    attribute of the export configuration. The association between exports and
-    &ganesha; services is done through the
-    <literal>conf-<replaceable>MINION_ID</replaceable></literal> objects. Each
-    service object contains a list of RADOS URLs for each export exported by
-    that service. An export block looks like the following:
-   </para>
-<screen>EXPORT
-{
-  Export_Id = 1;
-  Path = "/";
-  Pseudo = "/";
-  Access_Type = RW;
-  Squash = No_Root_Squash;
-  [...]
-  FSAL {
-    Name = CEPH;
-  }
-}</screen>
-   <para>
-    To create the &rados; object for the above export block, we first need to
-    store the export block code in a file. Then we can use the &rados; CLI tool
-    to store the contents of the previously saved file in a &rados; object.
+     Execute the following to create a <literal>nfs-ganesha</literal> service:
    </para>
 <screen>
-&prompt.cephuser;rados -p <replaceable>POOL</replaceable> -N <replaceable>NAMESPACE</replaceable> put export-<replaceable>EXPORT_ID</replaceable> <replaceable>EXPORT_FILE</replaceable>
+&prompt.cephuser;ceph orch apply nfs <replaceable>SERVICE_NAME</replaceable> --pool <replaceable>POOL_NAME</replaceable> --namespace <replaceable>NAMESPACE_NAME</replaceable>
```

Comment:
Do we not prefer the YAML file based deployment? `ceph orch apply -i <specfile>`

Reply:
@supriti @bk201 - could you confirm?

Reply:
Both work. A NFS service example can be found in this [page](https://docs.ceph.com/en/latest/mgr/orchestrator/#service-specification) (search for `A service of type nfs requires a pool name and may contain an optional namespace:`). It would be nice if we can also provide a link to the placement doc here.

Reply:
@bk201 will this cross link work or you? https://susedoc.github.io/doc-ses/master/single-html/ses-deployment/#cephadm-service-and-placement-specs

Reply:
@sebastian-philipp & @jschmid1 made  an agreement that we will recommend only the YAML based deployment in the docs for now

Reply:
I can update :) thanks for picking that up.

Reply:
> @bk201 will this cross link work or you? https://susedoc.github.io/doc-ses/master/single-html/ses-deployment/#cephadm-service-and-placement-specs

That will be nice! Can we still include an NFS service spec file as sample here?

Reply:
All done :) please review if you have a moment